.travis.yml

```diff
@@ -1,18 +1,23 @@
 language: ruby
 sudo: false
 cache: bundler
-script: 'bundle exec rake test:coverage --trace'
+script: 'bundle exec rubocop && bundle exec rake test:coverage --trace'
```

Comment:
@cllns For now I'd keep things simple and avoid yet another dep. Things may change after 1.0. 😉